git-helpers/src/lib.rs

```diff
@@ -18,3 +18,81 @@
 extern crate radicle_keystore as keystore;
 
 pub mod credential;
```

Comment:
Instead of putting this code here, I'd suggest:
* Adding `pub mod remote_helper` here
* Creating a `remote_helper.rs` file and moving this code into that file
* Renaming the main function to `run`
* Invoking the function from main with `remote_helper::run()`

Reply:
https://github.com/radicle-dev/radicle-link/pull/254/commits/1547c52c86e3d746a2403df07195f15406b0bb3b